doc/about.rst

```diff
@@ -0,0 +1,123 @@
+.. _about:
+
+##############################################################################
+About
+##############################################################################
```

Comment:
@bryevdv Thanks for feedback. Much appreciated. Disclaimer: Sphinx is still new to me.

Although, for headings I follow [Style guide for Sphinx-based documentations](http://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html#headings) regarding what markup to use at which level, I decided to tweak it with the 80-chars long heading.

My rationale is:

- If I change header title, I don't have to adjust the markup
- I *feel* like they stick out and visually separate sections with better geometric regularity
- They help to keep content aligned/wrapped at 80th column.
- I'm biased with C++ comments `/////////...` as separators in long files
- Whenever I looked at reStructuredText with short headers, I always wondered "Why not long?"

Enough explaining my feelings or interpretation of reST/Sphinx markup.

I do prefer to go for the generally preferred convention. If that is the short headers, I will switch.